cloudferrylib/os/actions/check_filter.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import utils as utl
+
+LOG = utl.get_log(__name__)
+
+
+class CheckFilter(action.Action):
+    def run(self, **kwargs):
+        """Check filter file and make sure all entries are present in source cloud.
+
+        """
+        search_opts = kwargs.get('search_opts', {})
+        search_opts_img = kwargs.get('search_opts_img', {})
+        search_opts_tenant = kwargs.get('search_opts_tenant', {})
+        image_resource = self.cloud.resources[utl.IMAGE_RESOURCE]
+        if search_opts_img and search_opts_img.get('images_list'):
+            images_list = search_opts_img['images_list']
+            for img_id in images_list:
+                LOG.debug('Filtered image id: {}'.format(img_id))
+                try:
+                    img = image_resource.glance_client.images.get(img_id)
+                    if img:
+                        LOG.debug('Image is OK')
```

Comment:
Please provide more meaningful message, for example: "Filter config check: Image ID %s is OK"